streams/src/test/java/org/apache/kafka/streams/integration/StreamStreamJoinIntegrationTest.java

```diff
@@ -384,7 +528,7 @@ public void testOuterRepartitioned() {
     public void testMultiInner() {
         STREAMS_CONFIG.put(StreamsConfig.APPLICATION_ID_CONFIG, appID + "-multi-inner");
 
-        final List<List<TestRecord<Long, String>>> expectedResult = Arrays.asList(
+        @SuppressWarnings("RedundantTypeArguments (explicit type arguments speedup compilation and analysis time)") final List<List<TestRecord<Long, String>>> expectedResult = Arrays.asList(
```

Comment:
For my own education: why do we need this, and why do we add the generic types below?

Reply:
Hmm.. I'm not sure where that suppression came from. Probably an accident, I will remove it.

The reason the explicit types are added below is because I was getting an error that type inference slows down if there are too many variadic arguments passed to a method with a generic that requires type inference. I forget if it was just my IDE complaining or if spotbugs was failing too but I figure adding the types in doesn't introduce any harm.